worker/static/show.htm

```diff
@@ -196,9 +196,9 @@ <h4 class="my-16 mx-4 text-3xl font-semibold flex justify-center text-center" la
             <template id="episode-pacing-legend-item">
                 <dt></dt>
                 <dd class="leading-relaxed truncate mr-2" lang="">Item</dd>
-                <div class="downloads-3 text-right font-mono opacity-50 mr-2 invisible md:visible" aria-labelledby="epl-3-day">123</div>
-                <div class="downloads-7 text-right font-mono opacity-50 mr-2 invisible md:visible" aria-labelledby="epl-7-day">123</div>
-                <div class="downloads-30 text-right font-mono opacity-50 mr-2 invisible md:visible aria-labelledby="epl-30-day"">123</div>
+                <div class="downloads-3 text-right font-mono opacity-50 mr-2 invisible md:visible" aria-describedby="epl-3-day">123</div>
+                <div id="epli-7" class="downloads-7 text-right font-mono opacity-50 mr-2 invisible md:visible" aria-describedby="epl-7-day epli-7">123</div>
```

Comment:
we can't put id's on these, id's should be unique in the document

the template here is repeated for every item in the grid

good catch on the double quote fix, I should have caught that - let me fix just that right now in another commit

Reply:
double quote fixed in 976e580b9b2fb1220831de9efe469ad9e3f02a9c

Reply:
My bad thanks